src/Lesson2.java

```diff
@@ -47,11 +56,11 @@ public static Boolean palindrome(String s) {
     }
 
     public static int[] randomArray(int numElemets) {
-        int max = 100;
+        int max = 250;
         int min = -100;
         int[] arrays = new int[numElemets];
         for (int i = 0; i < numElemets; i++) {
-            arrays[i] = (int) (Math.random() * ++max) + min;
+            arrays[i] = new Random().nextInt(max) + min;
```

Comment:
you're creating random object each time loop executes
Random object can be created as global variable and reused in all methods